### PR TITLE
Update to check for `isCustomProfile: 1` during PR validation (and fix existing incorrect profiles)

### PR DESCRIPTION
**Pull request by keveleigh**
## Overview
This PR extends the script added in #5632 to add a check for `isCustomProfile: 1` in all of our non-Examples assets.

## Changes
- Fixes: #5717, #5679

## Verification
I'll run the mrtk_PR pipeline to validate that this fails, then check in the fixed profiles and validate that this passes.

### CODE REVIEW

File: scripts/ci/validatecode.ps1 [hunk 1]
```diff
@@ -38,10 +38,28 @@ function CheckBooLang(
...
+    [string]$FileName,
+    [string[]]$FileContent,
+    [int]$LineNumber
+) {
+    <#
+    .SYNOPSIS
+        Checks if the given profile is marked as a custom profile.
+        Returns true if such a reference exists.
+    #>
+    if ($FileName -notmatch "Examples" -and $FileContent[$LineNumber] -match "isCustomProfile: 1") {
```
Comment (keveleigh) [inline scripts/ci/validatecode.ps1]: Do we care about profiles in the examples folders?

Comment (timGerken) [reply scripts/ci/validatecode.ps1]: @Kurtis, I'd say no.

**Comment by keveleigh** [comment]
Successful failure! 
image

**Comment by wiwei** [comment]
Yasssssssssssssssssss

**Pull request merged**